### PR TITLE
fix jetty dependencies

### PR DESCRIPTION
**Pull request by olamy**
- fix some dependencies

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -406,47 +407,71 @@
...
+                        <scope>import</scope>
+                    </dependency>
+                </dependencies>
+            </dependencyManagement>
             <dependencies>
                 <dependency>
                     <groupId>org.jboss.arquillian.container</groupId>
                     <artifactId>arquillian-jetty-embedded-11</artifactId>
-                    <version>1.0.0.CR4</version>
+                    <version>1.0.0.Final-SNAPSHOT</version>
```
Comment (olamy) [inline pom.xml:425]: because of https://github.com/arquillian/arquillian-container-jetty/pull/108

**Comment by hantsy** [comment]
@olamy Thanks. 
Hope there is a new release/milestone of aruqillian-jetty-11 will include your patch https://github.com/arquillian/arquillian-container-jetty/pull/108.

**Comment by olamy** [comment]
> @olamy Thanks. Hope there is a new release/milestone of aruqillian-jetty-11 will include your patch [arquillian/arquillian-container-jetty#108](https://github.com/arquillian/arquillian-container-jetty/pull/108).

temporary you can simply add the following dependency to your pom
```
    <dependency>
      <groupId>org.jboss.arquillian.testenricher</groupId>
      <artifactId>arquillian-testenricher-resource-jakarta</artifactId>
    </dependency>
```

**Comment by hantsy** [comment]
If I used the 1.0.0-CR4 and added `arquillian-testenricher-resource-jakarta`, some tests will be failed, it seems they are related to the CDI deps.

```log
[ERROR] com.example.it.GreetingServiceTest.should_create_greeting  Time elapsed: 0.163 s  <<< ERROR!
java.lang.NullPointerException
        at com.example.it.GreetingServiceTest.should_create_greeting(GreetingServiceTest.java:48)
...
[main] WARN org.eclipse.jetty.cdi.CdiServletContainerInitializer - Unable to decorate com.sun.faces.config.ConfigureListener@78854721
java.lang.IllegalStateException: Unable to access CDI
...
    
```

**Pull request merged**